woodwork/tests/accessor/test_statistics.py

```diff
@@ -673,8 +673,8 @@ def test_describe_accessor_method(describe_df):
                 "physical_type": expected_dtype,
                 "logical_type": ltype(),
                 "semantic_tags": {"custom_tag"},
-                "count": 6,
-                "nan_count": 2,
+                "count": 7,
+                "nan_count": 1,
```

Comment:
This behavior change is the only thing that gives me a little hesitation. We are now saying that `(np.nan, np.nan)` is not a null value, which is technically true I suppose. Before we would have replaced it with a single `nan` and recognized it as null.

Might not be worth the trouble it's worth to update the stats to handle this edge case, but now if a column has a lot of `(nan, nan)` tuples, we won't really identify that for the user as a potential highly null column.

Just wanted to mention in case anyone has a strong opinion on this behavior change.

Reply:
is there a way we can overload nan checking?

Reply:
added a check in describe to handle this case